docs/app/components/api-properties/api-properties.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'uxd-api-properties',
+    templateUrl: './api-properties.component.html',
+    styleUrls: ['./api-properties.component.less']
+})
+export class ApiPropertiesComponent {
+    @Input() title: string;
```

Comment:
One more thing - could you renamed this property? Having an attribute called "title" on the component causes the text to appear as a tooltip when hovering over any part of the table.

Reply:
This still needs to be addressed.